### PR TITLE
Feature/8.x.x service adapters

### PR DESCRIPTION
**Pull request by gonzalo-martinez-imt**

### CODE REVIEW

File: projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-ee.service.ts [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+import { share } from 'rxjs/operators';
+
+import { IAuthService } from '../../interfaces/auth-service.interface';
+import { IDataService } from '../../interfaces/data-service.interface';
+import { ServiceResponse } from '../../interfaces/service-response.interface';
+import { Util } from '../../util/util';
+import { BaseService } from '../base-service.class';
+
+@Injectable()
+export class OntimizeEEService extends BaseService implements IAuthService, IDataService {
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-ee.service.ts:13]: Ontimize services should extend from 'OntimizeBaseService'

File: projects/ontimize-web-ngx/src/lib/services/base-service.class.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+import { Codes } from '../util/codes';
+import { ServiceUtils } from '../util/service.utils';
+import { BaseServiceResponse } from './base-service-response.class';
+import { LoginStorageService } from './login-storage.service';
+import { OntimizeServiceResponseAdapter } from './ontimize/ontimize-service-response.adapter';
+import { OntimizeServiceResponseParser } from './parser/o-service-response.parser';
+
+export class BaseService implements IAuthService {
+
+  public kv = {};
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/src/lib/services/base-service.class.ts:22]: Those variables must not be here

File: projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-export.service.ts [hunk 3]
```diff
@@ -4,12 +4,12 @@ import { Router } from '@angular/router';
...
-import { ServiceUtils } from '../util/service.utils';
-import { LoginStorageService } from './login-storage.service';
+import { AppConfig } from '../../config/app-config';
+import { IExportService } from '../../interfaces/export-service.interface';
+import { Config } from '../../types/config.type';
+import { Codes } from '../../util/codes';
+import { ServiceUtils } from '../../util/service.utils';
+import { LoginStorageService } from '../login-storage.service';
 
 @Injectable({
```
Comment (manuel-cid) [inline projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-export.service.ts:14]: Remove `providedIn: 'root'`. This service is provided at component level in table export dialog

File: projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-export.service.ts [hunk 4]
```diff
@@ -4,12 +4,12 @@ import { Router } from '@angular/router';
...
   providedIn: 'root'
```
Comment (manuel-cid) [inline projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-export.service.ts:15]: Remove `providedIn: 'root'`, this service is provided at component level in table exportation dialog

**Pull request merged**